signer/signer.go

```diff
@@ -34,7 +34,7 @@ import (
 )
 
 // signingAgent is the unprotected header field used by signature.
-const signingAgent = "notation-go/1.2.0-rc.1"
```

Comment:
Please split the PR for dep bump up and the vote.

Reply:
This is notation-go library though, we are not voting from a PR. I think this update is more of a bump up?

Reply:
As discussed, we will vote from a PR for notation-go library in the future.